README.rst

```diff
@@ -397,3 +397,85 @@ the ``docs/build/html/index.html`` file
 
     $ cd docs
     $ make html
+
+
+POSTGRESQL WINDOWS INSTALLATION GUIDE
+-------------------------------------
+
+Download
+~~~~~~~~
+
+Download PostgreSQL via:
+https://www.postgresql.org/download/windows/
+
+Creating the Database and User
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create the postgresql database, in pgAdmin click "databases" and "create"
+and enter the desired characteristics of the database, including the owner.
+To create the user, similarly in pgAdmin, click "login roles" and "create"
+and enter the desired characteristics of the user. Ensure that it has 
+permisssion to login.
+
+Configuration
+~~~~~~~~~~~~~
+
+Installing requirements
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Ensure that C++ is installed -- if not, download from:
+https://www.microsoft.com/en-us/download/details.aspx?id=44266
+
+Ensure that setuptools are up-to-date by running:
+
+.. code:: bash
+
+    $ python -m pip install --upgrade pip setuptools
+
+Ensure that ez_setup is installed by running:
+
+.. code:: bash
+
+    $ pip install ez_setup
+
+Install requirements by running:
+
+.. code:: bash
+
+    $ pip install -r requirements.txt
+
+Configuration files
+^^^^^^^^^^^^^^^^^^^
+
+In $PATH\\data\pg_hba.conf, change the bottom few lines to read:
+
+# TYPE  DATABASE        USER            ADDRESS                 METHOD
```

Comment:
Can you put the config file contents in a [literal block](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)?